test-integration/helpers/ui/user.coffee

```diff
@@ -122,7 +129,8 @@ class User extends TestHelper
     # Going to the root URL while logged in will redirect to dashboard
     # which may redirect to the course page.
     @test.utils.verbose('Waiting until tutor-js page loads up')
-    @test.driver.wait(selenium.until.elementLocated(css: '#react-root-container .-hamburger-menu'))
```

Comment:
Hmm, I _think_ I used something other than `body` to make sure the page was one of tutor-js's (as opposed to a Gateway Error or tutor dashboard or accounts). But maybe it only adds confusion (and not necessary anymore?)